docs/items/clean-stick.md

```diff
@@ -0,0 +1,24 @@
+---
+id: clean-stick
+title: Bâton de nettoyage
+category: items
+description: Le bâton de nettoyage vous permet de supprimer les blocs inutiles de votre inventaire !
+icon: "textures/items/clean_stick.png"
+---
+___
+## Description
+
+Vous en avez marre de devoir enlever stack par stack les blocs inutiles que vous récupérez lorsque vous minez ? 
+Une utilisation du clean stick et le tour est joué, tous ces blocs seront instantanément supprimés de votre inventaire.
+
+### Caractéristiques:
+
+* ``Blocs supprimés :`` Blocs de Terre & blocs de Pierre
```

Comment:
Ce ne sont pas les seuls: 
- Granite
- Andésite
- Diorite
- Cobblestone et non la pierre